tests/robot-tests/tests/admin_and_public/data-catalogue.robot

```diff
@@ -0,0 +1,177 @@
+*** Settings ***
```

Comment:
- Shouldn't this be in the `bau` directory? We sign in as BAU so it makes sense to keep this with the other files in that directory.
- This should be `data_catalogue.robot` to follow file naming conventions - this also directly impacts on how the file displays in the test runner.